### ISSUE TITLE
accessibility feature request, collapse all comments of a comment thread

### ISSUE THREAD

**Issue by Hadi-1624** [QuantumBadger/RedReader#998]
Hi there, i am a blind user of the app, using screen reading software to  work with my phone.
Could we please have an option, in the long press menu to collapse all the comments of a single comment thread, so i could easily go to the next level 1 comment?
I can clarify more if needed.
Basically, when i'm scrolling down a comment and reading the level 2 and level 3 comments within, sometimes i don't want to follow that thread anymore, so i'm looking for a way to jump to the next level 1 comment, or collapsing the entire comments so i can go to the level 1 comment that is now collapsed, making it easy for me to jump to the next one.

**Comment by QuantumBadger** [comment]
Thanks for the suggestion!

If I add an accessibility action for "go to next top-level comment", would this solve the issue for you? It would be in the accessibility actions menu, and would automatically scroll to the next level 1 comment.

**Comment by Hadi-1624** [comment]
Thanks for your response!
by accessibility actions menu, do you mean the menu that you can enable in android's accessibility that puts a button near your screen's edge that stays around in all the apps?
We don't really use/turn it on, as screen reader users, as it gets in our way, though if there's no alternatives it's a good idea.
I was thinking perhaps it would be awesome if  "go to next top-level comment" was in the long press menu, somewhere near the up/down votes button so it would be quick to reach, would that be possible, perhaps a toggle in redreader's accessibility incase sighted users do not like such a thing?
Another idea that i had was that an option in redreader's accessibility that makes it so that clicking on a comment jumps to the next level 1 comment or collapses the entire thread, that way if single comments needed to be collapsed, we could use the long press menu.
Perhaps you have better ideas?
Thanks for the app, it is brilliant by the way.

**Comment by QuantumBadger** [comment]
If I remember correctly, the accessibility actions menu can be reached by swiping up then left in an upside-down L gesture, and then selecting "Actions". However I think in the TalkBack settings it's possible to make this gesture go directly to the actions menu.

I can add it to both the long press menu and the accessibility actions menu if that's helpful.

**Comment by Hadi-1624** [comment]
OH i see what you mean now! I use a third party screenr eader called Jieshuo and i think it also has the action menu as well. Sorry i didn't get it at  first, i'm a new android user.
That would be great, If you do ad it i will test it with both talkback and jieshuo
It would be the first and only action  in the menu once it comes up if i'm not mistaking, which would make it very quick to access.
Thanks!

**Comment by Hadi-1624** [comment]
> OH i see what you mean now! I use a third party screenr eader called Jieshuo and i think it also has the action menu as well. Sorry i didn't get it at first, i'm a new android user. That would be great, If you do ad it i will test it with both talkback and jieshuo It would be the first and only action in the menu once it comes up if i'm not mistaking, which would make it very quick to access. Thanks!

edit: i had another suggestion that is less crucial should i make another issue for it? I was looking for a way to make posts into one line (e.g, swipe once to go to next post)
